weave/trace_server/trace_server_interface.py

```diff
@@ -309,7 +337,7 @@ class CallsFilter(BaseModel):
 
 class SortBy(BaseModel):
     # Field should be a key of `CallSchema`. For dictionary fields
-    # (`attributes`, `inputs`, `outputs`, `summary`), the field can be
+    # (`attributes`, `inputs`, `output`, `summary`), the field can be
     # dot-separated.
     field: str  # Consider changing this to _FieldSelect
```

Comment:
I wont block here, but in general all comments for BaseModel should be Field descriptions :)

Reply:
yeah, we should take another pass. i'll clean up my interface